api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1859,6 +1859,97 @@ type AzurePlatformSpec struct {
 	// +immutable
 	// +required
 	SecurityGroupID string `json:"securityGroupID,omitempty"`
+
+	// ManagedIdentities contains the client IDs related to the managed identities needed for HCP control plane
```

Comment:
Should start with the serialised version of the field name
```suggestion
	// managedIdentities contains the client IDs related to the managed identities needed for HCP control plane
```